jasmine/jasmine-tests.ts

```diff
@@ -662,28 +662,34 @@ describe("jasmine.any", function () {
     });
 });
 
-describe("jasmine.objectContaining", function () {
-    var foo: any;
+describe("jasmine.objectContaining", () => {
+    interface fooType {
+        a: number;
+        b: number;
+        bar: string;
+    }
+    var foo: fooType;
 
-    beforeEach(function () {
+    beforeEach(() => {
         foo = {
             a: 1,
             b: 2,
             bar: "baz"
         };
     });
 
-    it("matches objects with the expect key/value pairs", function () {
-        expect(foo).toEqual(jasmine.objectContaining({
-            bar: "baz"
+    it("matches objects with the expect key/value pairs", () => {
+        expect(foo).toEqual(jasmine.objectContaining<fooType>({
+            bar: ''
```

Comment:
It might be good to leave a comment here explaining why
```ts
{bar: 'baz'} =~ {bar: ''}
```
as implied by the test.